### PR TITLE
Create UI for sign up state

### PR DESCRIPTION
**Pull request by DongTrieu1998**

### CODE REVIEW

**Comment by ghost** [comment]
Please provide some screenshots

**Comment by DongTrieu1998** [comment]
Screenshot 2024-09-18 000813


@thanhdat369

File: fudy/SignInScreen/SignUpPanel.qml [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+			textPressColor: FudyColor.fontColor1
+			backgroundColor: FudyColor.layer13
+			backgroundPressColor: FudyColor.layer13
+
+			text: qsTr("Sign in")
+		}
+	}
+
+	RowLayout {
+
```
Comment (ghost) [inline fudy/SignInScreen/SignUpPanel.qml]: why always empty line here

File: fudy/SignInScreen/SignUpPanel.qml [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+		SplitterItem {
+			width: 164
+			Layout.alignment: Qt.AlignVCenter
+			color: FudyColor.layer14
+		}
+	}
+
+	FudyButton {
+
```
Comment (ghost) [inline fudy/SignInScreen/SignUpPanel.qml]: -1

**Review by ghost — APPROVED** [review]

File: fudy/SignInScreen/SignUpPanel.qml (hunk 1 above)
Comment (DongTrieu1998) [reply fudy/SignInScreen/SignUpPanel.qml]: empty line was removed

File: fudy/SignInScreen/SignUpPanel.qml (hunk 2 above)
Comment (DongTrieu1998) [reply fudy/SignInScreen/SignUpPanel.qml]: empty line was removed

**Pull request merged**